src/quadpype/hosts/tvpaint/plugins/publish/extract_sequence.py

```diff
@@ -172,7 +184,7 @@ def process(self, instance):
                 mark_out,
                 filtered_layers,
                 ignore_layers_transparency,
-                apply_background,
+                self.apply_bg_back_command if self.settings_bg_color else "",
```

Comment:
Should be simply a bool, not a george script command since we won't use it in this method

`bool(self.settings_bg_color)` to convert the color array variable into a true/false bool